### PR TITLE
Use TextUitls.isEmpty() in the digest method

### PR DESCRIPTION
**Pull request by Smurph82**
I know this is not a big change but I was just getting my repo working and thought to push something simple to make sure it worked.

### CODE REVIEW

**Comment by koush** [comment]
Holy diff. I can't tell what changed!

File: Superuser/src/com/koushikdutta/superuser/util/Settings.java [hunk 1]
```diff
@@ -1,327 +1,327 @@
...
+        return Settings.getInstance(context).getString(KEY_PIN) != null;
+    }
+    
+    private static String digest(String value) {
+        // ok, there's honestly no point in digesting the pin.
+        // if someone gets a hold of the hash, there's really only like
+        // 10^n possible values to brute force, where N is generally
+        // 4. Ie, 10000. Yay, security theater. This really ought
+        // to be a password.
+        if (TextUtils.isEmpty(value))
```
Comment (laurelmay) [inline Superuser/src/com/koushikdutta/superuser/util/Settings.java:170]: Right around here + some. The code that GitHub automagically chose is not what I meant. I think Line 170 + a few lines after are where the differences are. That's where a difference with `TextUtils.isEmpty` is anyway.

**Comment by Smurph82** [comment]
Sorry about that I don't know why Git thought the whole file had changed. Could it be the EOL "End Of Line" as I was on Windows 7

Yes the only change was line 170.

**Comment by koush** [comment]
I'll do a git diff -w and see

**Comment by koush** [comment]
Got it:
https://github.com/koush/Superuser/pull/10/files?w=1#r3147267

**Pull request closed without merging**